#! /bin/sh /usr/share/dpatch/dpatch-run
## config.dpatch by Wolfgang Barth <wob@swobspace.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: changes default configs

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' nagvis-1.5.10~/etc/nagvis.ini.php-sample nagvis-1.5.10/etc/nagvis.ini.php-sample
--- nagvis-1.5.10~/etc/nagvis.ini.php-sample	2011-07-31 11:51:23.000000000 +0200
+++ nagvis-1.5.10/etc/nagvis.ini.php-sample	2011-09-26 16:24:19.000000000 +0200
@@ -108,7 +108,18 @@
 ; Path definitions
 [paths]
 ; absolute physical NagVis path
-;base="/usr/local/nagvis/"
+base="/usr/share/nagvis/htdocs/"
+cfg="/etc/nagvis/"
+var="/var/cache/nagvis/"
+images="/var/lib/nagvis/images/"
+icon="/var/lib/nagvis/images/iconsets/"
+shape="/var/lib/nagvis/images/shapes/"
+map="/var/lib/nagvis/images/maps/"
+headertemplate="/etc/nagvis/templates/header/"
+hovertemplate="/etc/nagvis/templates/hover/"
+contexttemplate="/etc/nagvis/templates/context/"
+gadget="/etc/nagvis/gadgets"
+
 ; absolute html NagVis path
 ;htmlbase="/nagvis"
 ; absolute html NagVis cgi path
@@ -280,7 +291,7 @@
 ; The statushost needs to be given in the following format:
 ;   "<backend_id>:<hostname>" -> e.g. "live_2:nagios"
 ;statushost=""
-;socket="unix:/usr/local/nagios/var/rw/live"
+socket="unix:/var/lib/icinga/rw/live"
 
 ; Example definition for a MySQL backend
 ; in this example the ID of the Backend is "ndomy_1" you can define another ID.
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' nagvis-1.5.10~/share/server/core/defines/global.php nagvis-1.5.10/share/server/core/defines/global.php
--- nagvis-1.5.10~/share/server/core/defines/global.php	2011-07-31 11:51:23.000000000 +0200
+++ nagvis-1.5.10/share/server/core/defines/global.php	2011-09-26 16:27:28.000000000 +0200
@@ -64,7 +64,7 @@
 define('DEBUGLEVEL', 6);
 
 // Path to the debug file
-define('DEBUGFILE', '../../../var/nagvis-debug.log');
+define('DEBUGFILE', '/var/log/nagvis-debug.log');
 
 // It is possible to define a second main configuration file
 // to pre-define some options in a file the user may not be
@@ -78,8 +78,8 @@
 // The last value wins.
 //
 // Path to the main configuration file
-define('CONST_MAINCFG', '../../../etc/nagvis.ini.php');
-define('CONST_MAINCFG_CACHE', '../../../var/nagvis.ini.php');
+define('CONST_MAINCFG', '/etc/nagvis/nagvis.ini.php');
+define('CONST_MAINCFG_CACHE', '/etc/nagvis/nagvis.ini.php');
 
 // Path to "site" main configuration file
 //define('CONST_MAINCFG_SITE', '../../../etc/nagvis-site.ini.php');
